plugins/package-managers/python/src/funTest/assets/projects/synthetic/pipenv/Pipfile

```diff
@@ -10,8 +10,8 @@ click = "==6.7"
 itsdangerous = "==0.24"
 Flask = "==1.0"
 Jinja2 = "==2.10.1"
-MarkupSafe = "==1.0"
+MarkupSafe = "==1.1"
 Werkzeug = "==0.15.3"
 
 [requires]
-python_version = "2"
+python_version = "3"
```

Comment:
Not sure how relevant it is, but this seems unrelated to what's described in the commit message. Could this stay at version 2?